lib/action.js

```diff
@@ -255,13 +259,13 @@ module.exports.actions = [
 			const selector = matches[2];
 			const eventType = matches[3];
 			try {
-				/* eslint-disable no-shadow, no-underscore-dangle */
-				await page.evaluate((selector, eventType) => {
-					const target = document.querySelector(selector);
+				/* eslint-disable no-underscore-dangle */
+				await page.evaluate((targetSelector, event) => {
```

Comment:
To keep this in line with the naming you've adopted across the other methods, I think this should be` desiredEvent`

```suggestion
				await page.evaluate((targetSelector, desiredEvent) => {
```